packages/bako-safe/src/DAPPWindow.ts

```diff
@@ -73,6 +75,9 @@ export class DAppWindow {
   }
 
   makeFrame(method: string, isSafari = false) {
+    //go to top
+    document.scrollingElement?.scrollTo(0, 0);
```

Comment:
Instead of scrolling to the top, you can fix the container on the screen size.

Reply:
this fix is ​​just to resolve this bug:


Uploading RPReplay_Final1718319302.mov…



- when the user is at the end of the page, even if you pass 100% of the screen height, we will have this problem.